src/xenia/kernel/xboxkrnl/xboxkrnl_strings.cc

```diff
@@ -49,6 +49,9 @@ enum FormatFlags {
   FF_IsWide = 1 << 9,
   FF_IsSigned = 1 << 10,
   FF_ForceLeadingZero = 1 << 11,
+  // IsWide gets used as an "is opposite encoding" flag, but this flag means
+  // it's definitely wide
+  FF_IsAlwaysWide = 1 << 12,
```

Comment:
Rename to `FF_IsWide`. And swap positions with the original `FF_IsWide` (now `FF_InvertWide`)-- including flag value.